server_addon/applications/server/settings.py

```diff
@@ -212,7 +212,13 @@ class ApplicationsAddonSettings(BaseSettingsModel):
         scope=["studio"]
     )
     only_available: bool = SettingsField(
-        True, title="Show only available applications")
+        True,
+        title="Show only available applications",
+        description="Enable to show only available applications in AYON Launcher"
+                    " for which the executable paths are found on the running machine."
+                    " This applies as an additional filter to the applications defined in a "
+                    " project's anatomy settings to ignore unavailable applications."
```

Comment:
```suggestion
        description="Enable to show only applications in AYON Launcher"
                    " for which the executable paths are found on the running machine."
                    " This applies as an additional filter to the applications defined in a "
                    " project's anatomy settings to ignore unavailable applications."
```
Should we remove "available" from the first sentence?

Reply:
I like your suggestion!